.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Motivation/Description of the PR
+
+Applicable helpers:
+
+[ ] Webdriver
+[ ] Puppeteer
+[ ] Nightmare
+[ ] REST
+[ ] Appium
+[ ] Protractor
+[ ] TestCafe
+
+- Description of this PR, which problem it solves
+- A link to the corresponding issue (if applicable).
+
+## Type of change
+
+- [ ] Breaking changes
+- [ ] New functionnality
+- [ ] Bug fix
+- [ ] Documentation change/update
+- [ ] Hot fix
```

Comment:
Not sure, that  "hot fix" should be here.

It's the same bug fix, but only urgent

Reply:
Yeah, it’s the point. Make the release to hot fix it. Bug fix can wait.